kicker/src/Collections/GameCollection.kt

```diff
@@ -0,0 +1,34 @@
+package com.kicker.Collections
+
+import com.kicker.Game
+import com.kicker.dbEnteties.Entity
+import kotlinx.coroutines.runBlocking
+import org.litote.kmongo.coroutine.CoroutineDatabase
+
+class GameCollection: Collection<Game> {
+    val database: CoroutineDatabase? = null
```

Comment:
you don't need coroutines here. same for runBlocking.
all operations here are synchronous.

if you opt in for async operations, you still need to remove runBlocking and get to know suspend function